src/resources/views/crud/fields/select2_from_ajax_multiple.blade.php

```diff
@@ -128,10 +129,15 @@ function bpFieldInitSelect2FromAjaxMultipleElement(element) {
                     delay: $ajaxDelay,
                     data: function (params) {
                         if ($includeAllFormFields) {
+                            // we trigger this event so that fields in need to parse their value before sent in some ajax request,
+                            // case of repeatable, can catch it and be sent with main form.
+                            form.trigger('backpack_field.parse_value', element);
+
                             return {
                                 q: params.term, // search term
                                 page: params.page, // pagination
-                                form: form.serializeArray() // all other form inputs
+                                form: form.serializeArray(), // all other form inputs
+                                ajax_trigger_input: {'row':element.attr('data-row-number'), 'element':$fieldName} //the names are purposedly big to avoid conflicts with any user inputs
```

Comment:
```suggestion
                                ajax_request_was_triggered_by: 
                                {
                                    'row': element.attr('data-row-number'), 
                                    'element': $fieldName
                                 }
```

@pxpm what do you think about renaming the variable like so? What bothered me was that `ajax_trigger_input` doesn't look like an array or object... you expect it to contain the name of the input, but then it returns something else.

Not sure this is any better. It certainly ain't prettier 😅